### PR TITLE
Reduce the complex of the controllers and add an adapter to compostiter.controller

### PR DESCRIPTION
**Pull request by luisdavidparra**

### CODE REVIEW

File: CONVERTER-SERVICE/src/controllers/videoConverter.controller.js [hunk 1]
```diff
@@ -11,52 +11,48 @@ accordance with the terms of the license agreement you entered into
...
-const Upload = require('../helpers/upload.helper');
 const Compress = require('../helpers/compress.helper');
-const VideoConverter = require('../models/videoConverter.model');
+const FileChecker = require('../helpers/fileChecker.helper');
+const ConverterChecker = require('../helpers/converterChecker.helper');
 const Converter = require('./converter.controller');
-const fs = require('fs');
 const { PORT, URL, URLBASE } = process.env;
 
 class VideoConverterController extends Converter {
```
Comment (DilanOjeda) [inline CONVERTER-SERVICE/src/controllers/videoConverter.controller.js:21]: Add a comment for the VideoConverterController class

File: CONVERTER-SERVICE/src/helpers/converterChecker.helper.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+const VideoConverter = require('../models/videoConverter.model');
+const ConverterException = require('../Exceptions/converter.exception');
+const libre = require('libreoffice-convert');
+
+class ConverterChecker {
+  
+  //Verifies the convert image process
+  static async convertImage(params) {
+    const {
+      Width,
```
Comment (DilanOjeda) [inline CONVERTER-SERVICE/src/helpers/converterChecker.helper.js]: Line 25 (Width) and 26 (Height) have to start with lowercase

File: CONVERTER-SERVICE/src/controllers/compositer.controller.js [hunk 3]
```diff
@@ -12,13 +12,13 @@ with Jalasoft.
 */
 
 const ImageCompositer = require('../models/imageCompositer.model');
+const FileChecker = require('../helpers/fileChecker.helper');
 const fs = require('fs');
 const path = require('path');
-const Upload = require('../helpers/upload.helper');
 const { PORT, URL, URLBASE } = process.env;
 
 class CompositerController {
```
Comment (DilanOjeda) [inline CONVERTER-SERVICE/src/controllers/compositer.controller.js:21]: Add a comment for the CompositerControllerclass

File: CONVERTER-SERVICE/src/controllers/docFileConverter.controller.js [hunk 4]
```diff
@@ -10,16 +10,14 @@ accordance with the terms of the license agreement you entered into
...
 
-const libre = require('libreoffice-convert');
-const ConverterException = require('../Exceptions/converter.exception');
-const Upload = require('../helpers/upload.helper');
+const FileChecker = require('../helpers/fileChecker.helper');
+const ConverterChecker = require('../helpers/converterChecker.helper');
 const Converter = require('./converter.controller');
-const fs = require('fs').promises;
 
 class docFileConverterController extends Converter {
```
Comment (DilanOjeda) [inline CONVERTER-SERVICE/src/controllers/docFileConverter.controller.js]: Add a comment for the docFileConverterController and also you could complete the word doc to document

File: CONVERTER-SERVICE/src/controllers/imageConverter.controller.js [hunk 5]
```diff
@@ -9,61 +9,44 @@ accordance with the terms of the license agreement you entered into
...
 
-const fs = require('fs');
-const Upload = require('../helpers/upload.helper');
-const ImageConverter = require('../models/imageConverter.model');
+const FileChecker = require('../helpers/fileChecker.helper');
+const ConverterChecker = require('../helpers/converterChecker.helper');
 const Converter = require('./converter.controller');
 const { PORT, URL, URLBASE } = process.env;
 
 class ImageConverterController extends Converter {
```
Comment (DilanOjeda) [inline CONVERTER-SERVICE/src/controllers/imageConverter.controller.js:18]: Add a comment for the ImageConverterController class

File: CONVERTER-SERVICE/src/helpers/converterChecker.helper.js [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+*/
+
+const fs = require('fs');
+const fsPromise = require('fs').promises;
+const ImageConverter = require('../models/imageConverter.model');
+const VideoConverter = require('../models/videoConverter.model');
+const ConverterException = require('../Exceptions/converter.exception');
+const libre = require('libreoffice-convert');
+
+class ConverterChecker {
```
Comment (DilanOjeda) [inline CONVERTER-SERVICE/src/helpers/converterChecker.helper.js:21]: Add a comment for the ConverterChecker class

File: CONVERTER-SERVICE/src/helpers/converterChecker.helper.js [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+      isActiveNegative == 'true'
+    );
+
+    await convertImage.convert();
+  }
+
+  //Verifies the convert video process
+  static async convertVideo(params, req) {
+    const { fps, imageSize, saveVideoPath, saveImagesPath } = params;
+
```
Comment (DilanOjeda) [inline CONVERTER-SERVICE/src/helpers/converterChecker.helper.js]: a line break should not be at this line

File: CONVERTER-SERVICE/src/helpers/converterChecker.helper.js [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+      imageSize
+    );
+
+    await convertVideo.convert();
+  }
+
+  //Verifies the convert doc process
+  static async convertDocFile(params) {
+    const { ext, inputPath, outputPath } = params;
+
```
Comment (DilanOjeda) [inline CONVERTER-SERVICE/src/helpers/converterChecker.helper.js]: a line break should not be at this line

File: CONVERTER-SERVICE/src/helpers/fileChecker.helper.js [hunk 9]
```diff
@@ -12,26 +12,22 @@ with Jalasoft.
 
 const InvalidFileException = require('../Exceptions/invalidFile.exception');
 
-class Upload {
-  
+class FileChecker {
```
Comment (DilanOjeda) [inline CONVERTER-SERVICE/src/helpers/fileChecker.helper.js:16]: Add a comment for the FileChecker class

File: CONVERTER-SERVICE/src/helpers/fileChecker.helper.js [hunk 10]
```diff
@@ -12,26 +12,22 @@ with Jalasoft.
...
+
   //Receives the video from the server and saves it in the "files" local file
-  static uploadVerified(params, type) {
-    let { originalname, mimetype } = params;
+  static uploadChecker(params, type) {
+    let { mimetype } = params;
 
-    if (type === 'DOC') mimetype = 'file/docx';
+    if (mimetype.endsWith('document')) mimetype = 'file/docx';
```
Comment (DilanOjeda) [inline CONVERTER-SERVICE/src/helpers/fileChecker.helper.js]: Try to use the format
if (condition) {
  
}

Comment (luisdavidparra) [reply CONVERTER-SERVICE/src/helpers/fileChecker.helper.js]: All done

**Review by DilanOjeda — APPROVED** [review]
Good job

**Pull request merged**